src/config.h

```diff
@@ -149,7 +154,7 @@ sds RSConfig_GetInfoString(const RSConfig *config);
 #define RS_DEFAULT_CONFIG                                                                       \
   {                                                                                             \
     .concurrentMode = 1, .extLoad = NULL, .enableGC = 1, .minTermPrefix = 2,                    \
-    .maxPrefixExpansions = 200, .queryTimeoutMS = 500, .timeoutPolicy = TimeoutPolicy_Return,   \
+    .maxPrefixExpansions = 800, .queryTimeoutMS = 500, .timeoutPolicy = TimeoutPolicy_Return,   \
```

Comment:
why you did not add prefixExpansionNoError default value here?

Reply:
Because we didn't merge that other PR yet :)